custom_fractional_fee.go

```diff
@@ -7,19 +7,21 @@ import (
 
 type CustomFractionalFee struct {
 	CustomFee
-	Numerator     int64
-	Denominator   int64
-	MinimumAmount int64
-	MaximumAmount int64
+	Numerator        int64
+	Denominator      int64
+	MinimumAmount    int64
+	MaximumAmount    int64
+	AssessmentMethod FeeAssessmentMethod
 }
 
 func customFractionalFeeFromProtobuf(fractionalFee *proto.FractionalFee, fee CustomFee) CustomFractionalFee {
 	return CustomFractionalFee{
-		CustomFee:     fee,
-		Numerator:     fractionalFee.FractionalAmount.Numerator,
-		Denominator:   fractionalFee.FractionalAmount.Denominator,
-		MinimumAmount: fractionalFee.MinimumAmount,
-		MaximumAmount: fractionalFee.MaximumAmount,
+		CustomFee:        fee,
+		Numerator:        fractionalFee.FractionalAmount.Numerator,
+		Denominator:      fractionalFee.FractionalAmount.Denominator,
+		MinimumAmount:    fractionalFee.MinimumAmount,
+		MaximumAmount:    fractionalFee.MaximumAmount,
+		AssessmentMethod: FeeAssessmentMethod(fractionalFee.NetOfTransfers),
```

Comment:
Shouldn't this  use `valueOf`? If this works without using `valueOf` then we should just remove that function.